fpga_interchange/fpga_interchange_archdefs.h

```diff
@@ -0,0 +1,87 @@
+#include <cstdint>
```

Comment:
I'm not immediately sure why both archdefs.h and fpga_interchange_archdefs.h are needed?

Reply:
This was an old file, I'll remove.